package.json

```diff
@@ -42,7 +42,8 @@
     "dev": "nf start -w --procfile ./dev-tools/Procfile.dev",
     "dev-nowrap": "nf start --trim 500 --procfile ./dev-tools/Procfile.dev --env ./__test__/e2e/.env.e2e",
     "debug-server": "nf start -w --procfile ./dev-tools/Procfile-debug-server.dev",
-    "dev-other": "nf start -w --procfile ./dev-tools/Procfile-other.dev"
+    "dev-other": "nf start -w --procfile ./dev-tools/Procfile-other.dev",
```

Comment:
other.dev doesn't seem to exist, so please remove.

Reply:
it's not `other.dev`, it's `Procfile-other.dev` and it's definitely in the repository